cli/test/org/partiql/cli/functions/ReadFileTest.kt

```diff
@@ -151,4 +151,64 @@ class ReadFileTest {
 
         assertEquals(ion.singleValue(expected), actual)
     }
+
+    @Test
+    fun readExcelCsvFile() {
+        writeFile("simple_excel.csv", "title,category,price\nharry potter,book,7.99")
+
+        val args = listOf("\"${dirPath("simple_excel.csv")}\"", "{type:\"excel_csv\", header:true}").map { it.exprValue() }
+
+        val actual = function.call(env, args).ionValue
+        val expected = "[{title:\"harry potter\",category:\"book\",price:\"7.99\"}]"
+
+        assertEquals(ion.singleValue(expected), actual)
+    }
+
+    @Test
+    fun readPostgreCsvFile() {
+        writeFile("simple_postgre.csv", "id,name,balance\n1,Bob,10000.00")
```

Comment:
typo: 'simple_postgre.csv' -> 'simple_postgresql.csv'

```suggestion
    fun readPostgreSQLCsvFile() {
        writeFile("simple_postgresql.csv", "id,name,balance\n1,Bob,10000.00")
```